internal/cmd/beta/kms/key/list/list.go

```diff
@@ -0,0 +1,147 @@
+package list
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/goccy/go-yaml"
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-cli/internal/cmd/params"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/kms/client"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/tables"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/utils"
+	"github.com/stackitcloud/stackit-sdk-go/services/kms"
+)
+
+const (
+	keyRingIdArg = "KEYRING_ID"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	KeyRingId string
+}
+
+func NewCmd(params *params.CmdParams) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   fmt.Sprintf("list %s", keyRingIdArg),
+		Short: "List all KMS keys",
+		Long:  "List all KMS keys inside a key ring.",
+		Args:  args.SingleArg(keyRingIdArg, utils.ValidateUUID),
+		Example: examples.Build(
+			examples.NewExample(
+				`List all KMS keys for the key ring "xxx"`,
+				"$ stackit beta kms key list xxx"),
```

Comment:
```suggestion
				"$ stackit beta kms key list --keyring-id xxx"),
```

To stay consistent withour current implementation, the keyring ID would be a flag instead of an argument here. See example below for reference.

https://github.com/stackitcloud/stackit-cli/blob/14481e42f57b366c1310e8027c41604b75473351/internal/cmd/mongodbflex/user/list/list.go#L38-L51